abtesting/README.md

```diff
@@ -6,22 +6,30 @@ behaviors and exports the experiment results into Firebase Analytics. This
 sample demonstrates using an A/B test to test multiple color schemes within
 an app.
 
-## Quickstart Setup
+## Create Experiment
 
-In Firebase Console, create a new A/B test with any name. In step two under
-'Variants', click the 'Add Parameter' button and create a parameter named
-`color_scheme`. Set its default value in the control group to `default` and
-its value in 'Variant A' to `dark`. In the third step, select any goal metric.
+In Firebase Console's A/B Testing section, click the 'Create experiment' button
```

Comment:
Backquotes around `Create Experiment` and other button and option selection names below as well.

Reply:
Done ✅